ui/components/Tag/index.js

```diff
@@ -1,6 +1,6 @@
 import classnames from 'classnames';
 import styles from './style.module.scss';
 
-export default function Tag({ children }) {
-  return <span className={classnames('nhsuk-tag', styles.tag)}>{ children }</span>
+export default function Tag({ children, classes }) {
+  return <span className={classnames('nhsuk-tag', styles.tag, styles.[classes])}>{ children }</span>
```

Comment:
does this work? wouldn't it need to be `styles[classes]`?

Reply:
That's what worked from trial (and error!) which I'm assuming means `classnames` expects a dot accessor and resolves the key of `classes` before accessing? I'm confused too :)

Reply:
Switched it! Still works? Will probably drop this for a js mapping of status to class name e.g.

```
const statusMap = {'active': 'nhsuk-tag--blue',
'draft': 'nhsuk-tag--grey',
'deprecated': 'nhsuk-tag--orange',
'retired': 'nhsuk-tag--red'};

return statusMap(status)
```